deploy/kustomize/statefulset-action-demo.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: github-runner-action-demo
+spec:
+  serviceName: github-runner-action-demo
+  replicas: 2
+  selector:
+    matchLabels:
+      app: github-runner-action-demo
+  template:
+    metadata:
+      labels:
+        app: github-runner-action-demo
+    spec:
+      containers:
+      - name: agent
+        image: swaglive/github-runner:2.163.1-ubuntu18.04
+        imagePullPolicy: Always
```

Comment:
Probably don't need this since its tagged with version

Reply:
Yes. It's for debugging. I'll remove it.